### PR TITLE
fix(installation): added composer.json and registartion.php

### PR DESCRIPTION
**Pull request by omryYotpo**

### CODE REVIEW

File: Yotpo/Yotpo/composer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "yotpo/module-yotpo",
+  "description": "N/A",
```
Comment (vladshub) [inline Yotpo/Yotpo/composer.json]: @omryYotpo please add a proper description to our module... !fix

File: Yotpo/Yotpo/composer.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "yotpo/module-yotpo",
```
Comment (vladshub) [inline Yotpo/Yotpo/composer.json]: @omryYotpo does the name is the name of the module in the file system? or can we put a proper name?
and it seems that we can do a composer package to this and distribute this over compose.io

**Comment by dmitri86git** [comment]
@omryYotpo  There is a bug when I trying to approve an order.
It gives me this error:
screen shot 2015-12-14 at 12 50 42 pm 3
To fix it please go to Yotpo\Yotpo\Helper/ApiClient.php line 14 and change Resource to ResourceModel.

After doing that you'll see one more alert: 
screen shot 2015-12-14 at 12 54 50 pm 3

File: Yotpo/Yotpo/composer.json (hunk 2 above)
Comment (omryYotpo) [inline Yotpo/Yotpo/composer.json]: I think we can use a composer repo.
about the name i'm not sure if it's required- but it sure is a convention..it's like that in all the modules i saw.

File: Yotpo/Yotpo/composer.json (hunk 1 above)
Comment (omryYotpo) [inline Yotpo/Yotpo/composer.json]: !fixed  in a newer commit

File: Yotpo/Yotpo/composer.json (hunk 2 above)
Comment (vladshub) [inline Yotpo/Yotpo/composer.json]: OK

**Pull request merged**